macros/core/PGML.pl

```diff
@@ -1562,20 +1538,27 @@ sub Align {
 );
 
 sub List {
-	my $self = shift;
-	my $item = shift;
-	my $list = $bullet{ $item->{bullet} };
+	my ($self, $item, $block, $state) = @_;
+	my $list   = $bullet{ $item->{bullet} };
+	my $margin = '0';
+	if (defined $block->{stack}[ $state->{i} ] && $block->{stack}[ $state->{i} ]{type} eq 'par') {
+		$margin = '0 0 1em 0';
```

Comment:
Again, if you want to reduce characters, leave of the final ` 0`, which is implied by the second `0`.